custom-routes-vis/src/main/webapp/script.js

```diff
@@ -319,11 +308,13 @@ function createRouteFromCoordinates(
 
   // Select the first route as default.
   if (routeNum === 0) {
-    selectRouteDisplayDetails(0, totalDurationSec, totalDistanceMeters);
+    selectRouteDisplayDetails(0, totalDurationSec, totalDistanceMeters, 
+        routeToken);
```

Comment:
I think it should be something like this?
Could you check the documentation on this?

```suggestion
    selectRouteDisplayDetails(
        0, totalDurationSec, totalDistanceMeters, routeToken);
```

Reply:
I think this is correct. https://google.github.io/styleguide/jsguide.html#formatting-where-to-break